decentraland/unity-explorer
# Feat: camera reel - navmap integration

## What does this PR change?



Implements the place gallery into the appropriate navmap section, showing all public reels taken in a specific place.

## How to test the changes?



1. Launch the explorer with all reels base urls pointing to `zone`
2. Open the navmap
3. Click on a parcel
4. Switch to the `PHOTOS` section
5. Click on any reels and verify it opens the photo detail view

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
